client/src/extension.ts

```diff
@@ -30,6 +30,13 @@ var terminal;
 const fs = require('fs');
 const url = require('url');
 
+const FILE_ASSOCIATIONS:string = 'files.associations';
+const REACH_FILE_EXT:string = '*.rsh';
```

Comment:
This is really more than an extension, but a wildcard selector. Maybe `REACH_FILE_SELECTOR` is a more appropriate name